dockerImageUpdate.js

```diff
@@ -4,37 +4,53 @@
 // stack.
 //
 // Use when a new docker image is available for our services.
+// Works with docker swarm or podman compose
 //
 require("dotenv").config();
+/** @type {{ version: string, services: Object.<string, string> }} */
 const version = require("./version");
 const { exec } = require("child_process");
 const path = require("path");
 const fs = require("fs").promises;
+const migrationContainer = "update_script_db_migrations"; // ref to clean up the migration container
 
 // const cwd = process.cwd();
 
-const stack = ` ${process.env.STACKNAME}`;
-const stackNoSpace = stack.replace(" ", "");
+const stack = process.env.STACKNAME;
```

Comment:
I'm not very familiar with this, but are we no longer worried about someone putting " " in the STACKNAME?

Reply:
We don't do any checks elsewhere for spaces in the `STACKNAME`, so I 'd be really surprised if someone got here with that being the case. From what I can tell the, we first used the local `stack` variable with a space proceeding the stackname, my guess is this made composing commands easier at the time. At some point we added `stackNoSpace` which doesn't have the space preceding it, again my guess is this was for convienence composing commands in a slightly different way. Anyways at this point we can just the `STACKNAME` without adding a space in all cases.